chain/types/bigint_test.go

```diff
@@ -82,7 +82,7 @@ func TestSizeStrUnitsSymmetry(t *testing.T) {
 	s := rand.NewSource(time.Now().UnixNano())
 	r := rand.New(s)
 
-	for i := 0; i < 1000000; i++ {
+	for i := 0; i < 1000; i++ {
```

Comment:
This should be at least `10000` to be useful
```suggestion
	for i := 0; i < 10000; i++ {
```